src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -196,11 +197,48 @@ export class CreatibutorsModal extends Component {
     }
   };
 
+
+  makeIdEntry = (identifier) => {
+
+    let icon = null;
+    let link = null;
+    let linkTitle = null;
+
+    if (identifier.scheme == "orcid") {
+      icon = "/static/images/orcid.svg";
+      link = "https://orcid.org/" + identifier.identifier;
+    } else if (identifier.scheme == "gnd") {
+      icon = "/static/images/gnd-icon.svg";
+      link = "https://d-nb.info/gnd/" + identifier.identifier;
+    } else if (identifier.scheme == "ror") {
+      icon = "/static/images/ror-icon.svg";
+      link = "https://ror.org/"+identifier.identifier;
+    } else {
+      return (<>{identifier.scehme}: {identifier.identifier}</>)
+    }
+
+    return (
+      <>
+        <Image
+          src={icon}
+          className="inline-id-icon"
+          verticalAlign="middle"
+          key={identifier.identifier}
+        />
+        {identifier.identifier}
+          <a
```

Comment:
could you add a little margins between the image, identifiers and the external link Icon ? you could use the class `mr-5` on the `Image` and `ml-5` on the `<a` tag. The ocntent looks a little bit "packed" and I think we could use some negative space

Reply:
No problem - I just repeated the existing element which was already tightly packed, and the duplication makes it worse. Do you think it actually needs the external link icon? In other places the identifier is a link without this. 

(I'm also converting the idString's to <span> elements because react was complaining they needed a key attribute...)

Reply:
![image](https://user-images.githubusercontent.com/7934352/173561516-fed8e327-984e-4208-bc2b-69c03c05e10c.png)

I think that's better?

Reply:
I think we need some kind of a separator between the icons and orcid/ROR logos, otherwise I am not sure where one identifier entry starts and where it ends (fe. the orcid could be easily understood as a part of the previous entry). Would you mind throwing a separator in between, f.e. a semicolon?

Reply:
I am totally fine also with removing the external link icon, to clear out the display

Reply:
Agree, link icon removed and separators inserted:

![](https://cdn.discordapp.com/attachments/878276463990800474/986237363531440178/Screenshot_2022-06-14_at_12.54.46.png)